### PR TITLE
Gp 12113/batch process runner

### PR DESCRIPTION
**Pull request by dcumberland**
Sets up pod group for running the spring batch application

### CODE REVIEW

**Comment by dcumberland** [comment]
It's probably clearer if you review just this commit: https://github.com/Git-Prime/gphelm-chart-repo/pull/34/commits/3509b959a76fca7bc2e368f609cf7bca4d781a2d

File: charts/gitprime-app/v4.0.0/questions.yml [hunk 1]
```diff
@@ -494,6 +494,71 @@ questions:
...
 
+
+  # Batch Worker Settings
+  - variable: dataPipeline.batchWorker.replicaCount
+    type: string
+    required: true
+    group: "Data Pipeline / Batch Workers"
+    label: Number of Pods for Batch Workers
+    description: The number of pods to deploy for the batch workers.
+    default: 1
```
Comment (dcumberland) [inline charts/gitprime-app/v4.0.0/questions.yml]: I *think* we only ever want 1 of the batch workers.  @ntaylor-gitprime should we hard code the helm chart to only ever allow a single pod of the batch worker?

Comment (nate-taylor1) [reply charts/gitprime-app/v4.0.0/questions.yml]: Yes, I can't think of a scenario where we would want multiple JVMs running the same batch processes.

**Review by nate-taylor1 — APPROVED** [review]
LGTM other than the hardcoding of the batch worker pod instances to 1.

**Pull request merged**